pages/home/home.tsx

```diff
@@ -0,0 +1,34 @@
+import type { NextPage } from "next";
```

Comment:
Move this file according to routing rules in this case i can access the page via this route localhost:3000/home/home which is improper behavior
Check [Next doc](https://nextjs.org/docs/routing/introduction) for more info

This and related files sould be src/modules/homepage